receiver/dcgmreceiver/client.go

```diff
@@ -328,9 +328,11 @@ func (client *dcgmClient) collect() (time.Duration, error) {
 		if err := isValidValue(fieldValue); err == errBlankValue {
 			// Blank values are expected at startup.
 			continue
+		} else if err == errNotSupported {
+			client.issueWarningForFailedQueryUptoThreshold(dcgmName, 1, fmt.Sprintf("Field '%s' is not supported. Metric '%s' will not be collected", dcgmName, dcgmName))
```

Comment:
The `Metric '%s' will not be collected` part is redundant (in the profiling metrics as well). But I can fix that after we merge this PR…